server/WebAPI/Services/UserService.cs

```diff
@@ -53,6 +54,12 @@ public void CreateUser(string email, string password)
       _userRepository.CreateUser(user);
     }
 
+    public void CreateOauthUser(string email)
+    {
+      var user = new User {Email = email, IsOauth = true};
```

Comment:
if google gives something more then just email, (e.g. name, google_id) you can save it into user profile as well